site/src/ThemeSelector.jsx

```diff
@@ -1,39 +1,60 @@
 import PropTypes from "prop-types";
 import { useState, useEffect } from "react";
 import LayerIcon from "./icons/icon-layers.svg?react";
-import "./ThemeSelector.css"; 
+import "./ThemeSelector.css";
 
 function ThemeSelector({ interactiveLayers }) {
-
   const [places, setPlaces] = useState(true);
   const [buildings, setBuildings] = useState(true);
   const [transportation, setTransportation] = useState(true);
 
-  
   useEffect(() => {
-
     let layers = [];
 
-    if (places) layers.push('places');
-    if (buildings) layers.push('buildings');
-    if (transportation) layers.push('transportation');
+    if (places) layers.push("places");
+    if (buildings) layers.push("buildings");
+    if (transportation) layers.push("transportation");
     interactiveLayers(layers);
   }, [buildings, places, transportation, interactiveLayers]);
 
   return (
-    <div className="dropdown dropdown--hoverable theme-selector">
+    <div className="dropdown dropdown--hoverable theme-selector tour-layers">
```

Comment:
Only contentful change. This was added so the tour has a 'target'. All else is formatting changes.